Tests/Blast/xml_2226_tblastn_007_err.xml

```diff
@@ -0,0 +1,23 @@
+this is an empty file to test the method NCBIXML.handle()
```

Comment:
Precisely.
I could have gone with a random string like "aksjdhaskjdnnJadsfaddNASDLNAIDIN", but I choose this. 

Do you think I should change it?